editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -260,6 +270,10 @@ export async function renderTestEditorWithModel(
     )
   })
 
+  if (awaitFirstDomReport === 'await-first-dom-report') {
```

Comment:
have you tried just always awaiting for this, and running all tests? I can't come up with any test that should use `renderTestEditorWithModel` and not await the first dom report

Reply:
Yep. The non-browser tests failed (or at least some of them did).